crates/schema/tests/validate_integration.rs

```diff
@@ -18,8 +18,11 @@ const TEST_TABLE_ID: TableId = TableId(1337);
 #[allow(clippy::disallowed_macros)] // LET ME PRINTLN >:(
 fn validate_module(module_name: &str) {
     let module = CompiledModule::compile(module_name, CompilationMode::Debug);
-    let raw_module_def: RawModuleDefV8 =
+    let raw_module_def: RawModuleDef =
         extract_descriptions(module.path()).expect("failed to extract module descriptions");
+    let RawModuleDef::V8BackCompat(raw_module_def) = raw_module_def else {
+        panic!("no more v8")
+    };
```

Comment:
So both C# and rust are patched to produce v9 now?

Reply:
No, not yet - the `panic!()` will happen when v8 is no longer being produced.